src/software/visualizer/drawing/navigator.h

```diff
@@ -1,7 +1,27 @@
 #pragma once
 
+#include <QtWidgets/QGraphicsScene>
+
 #include "software/ai/navigator/navigator.h"
+#include "software/ai/navigator/obstacle/obstacle.h"
+#include "software/visualizer/drawing/colors.h"
 #include "software/visualizer/drawing/draw_functions.h"
+#include "software/visualizer/drawing/geom.h"
+#include "software/visualizer/geom/geometry_conversion.h"
 
-// TODO: comment
+/**
+ * Gets the draw function for drawing the navigator
```

Comment:
uber nit: What does it mean to "draw a navigator"? Is this drawing the paths per robot? Can we quickly add a comment here saying what gets drawn?

Reply:
It's somewhat undefined, because the function can draw as much of the navigator as possible. For now, it's paths and obstacles, but we could draw more.